### PR TITLE
Remove Dead and Deprecated Web Services

### PR DESCRIPTION
**Pull request by grahamc**
- repoman 404's so remove the svn redirect
 - remove the long-commented svn subservice
 - stop serving the wiki, remove related redirects, and remove related pgsql config

I don't mean this to be a forever thing, I think having the wiki in a middle phase between "is old and out of date and not fixable" and "we're wanting to fix it soon(?)" is the worst option. Nobody sees the "being shut down" message.

cc @copumpkin

### CODE REVIEW

File: nixos-org/webserver.nix [hunk 1]
```diff
@@ -308,13 +221,8 @@ in
       synchronous_commit = off
     '';
     authentication = mkOverride 10 ''
-      local mediawiki all ident map=mwusers
       local all       all ident
     '';
-    identMap = ''
-      mwusers root   mediawiki
-      mwusers wwwrun mediawiki
-    '';
```
Comment (edolstra) [inline nixos-org/webserver.nix]: Since Postgres is only used for the wiki, it can be removed entirely.

Comment (grahamc) [inline nixos-org/webserver.nix]: 👍 done

**Comment by copumpkin** [comment]
Rather than take down the wiki webserver entirely, can you just replace it with a landing page saying something like:

> The NixOS wiki is currently down. Please check <manual URLs> <@domenkozar's cookbooks> for documentation in the meantime, and follow https://github.com/NixOS/nixpkgs/issues/22599 for updates on the wiki itself.

It seems friendlier than just making it disappear when it's by far the greatest target of Google links for most Nix searches.

**Comment by grahamc** [comment]
I didn't expect you to have such a positive response! I thought for sure you'd just ask for another week or something :)

**Comment by copumpkin** [comment]
I agree with you that we're in the worst possible situation right now, and I have very little time for anything much, so I don't want to leave things shitty due to my lack of time.

**Comment by copumpkin** [comment]
@edolstra can you just keep a backup of the wiki data lying around for a while? Not saying it needs to be publicly accessible, but I'd hate to lose the info forever.

**Comment by grahamc** [comment]
I'm looking in to making an export, there is https://nixos.org/wiki/Special:Export but it is a bit annoying since it is per-category.

**Comment by grahamc** [comment]
Ok, I copied all the pages from https://nixos.org/wiki/Special:AllPages and pasted them in to https://nixos.org/wiki/Special:Export and made an export of all pages with all versions, available for a limited time here: http://next.gsc.io/Nix+Wiki-20170503130016.xml

@edolstra can you mirror this export to somewhere on nixos.org, so it can be linked to from the "down" page?

**Comment by grahamc** [comment]
This PR now has a redirect and a second PR adds the landing page: https://github.com/NixOS/nixos-homepage/pull/138

I think this and the other are ready.

**Comment by grahamc** [comment]
Rebased on master after the other PR merged. Ready to be merged.

**Pull request merged**